server/views/upload-document.html

```diff
@@ -46,31 +49,54 @@ <h2 id="helpTextSubHeading" class="govuk-heading-m">Upload file</h2>
             <li>smaller than {{ maximumFileSize }}MB</li>
           </ul>
 
-        {{ govukFileUpload({
-          attributes: {
-            accept: accept
-          },
-          id: "files",
-          name: "files",
-          errorMessage: fieldErrors['files']
-        }) }}
+          {{ govukFileUpload({
+            attributes: {
+              accept: accept
+            },
+            id: "files",
+            name: "files",
+            errorMessage: fieldErrors['files']
+          }) }}
+        </div>
+
+        <div id="uploading1" style="display:none">
+          <h2 id="uploadingHelpTextSubHeading" class="govuk-heading-m">Uploading file</h2>
+
+          <p id="uploadingHelpText" class="govuk-body">Your file is being processed<br>Please wait...</p>
+
+          <!-- style: large -->
+          <div class="ispinner ispinner-large">
+            <div class="ispinner-blade"></div>
```

Comment:
Avoid repeated code - the following "for" loop should work:

{% for i in range(0, 7) %}
  <div class="ispinner-blade"></div>
{% endfor %}

Reply:
I'm not sure of the syntax though, these docs:
https://mozilla.github.io/nunjucks/templating.html#range-start-stop-step

Suggest:
{% for i in range(0, 5) **-**%}
  {{ i }},
{%**-** endfor %}